buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -40,7 +40,9 @@ fun Project.createStackDeployTask(profile: String) {
                         // so we create it here only in dev profile
                         """|$it
                            |  mysql:
-                           |    image: mysql:8.0.20
+                           |    image: mysql
```

Comment:
What about tag?

Reply:
Which one do we want? `latest`?

Reply:
Did you try `8.0.28-oracle` on M1 without explicit `platform`? If it works, we can use that tag.

Reply:
Yes, it works, will remove `platform` and use tag `8.0.28-oracle`